wait/awaitility.go

```diff
@@ -121,7 +121,7 @@ func (a *SingleAwaitilityImpl) WaitForKubeFedClusterConditionWithName(name strin
 			a.T.Logf("found %s KubeFedCluster", name)
 			return true, nil
 		}
-		a.T.Logf("waiting for %s KubeFedCluster having the expected condition", name)
+		a.T.Logf("waiting for %s KubeFedCluster having the expected condition (expected: %v vs actual: %v)", name, condition, cluster.Status.Conditions)
```

Comment:
I think that it's better to use `%+v` as it prints also the names of the parameters
```suggestion
		a.T.Logf("waiting for %s KubeFedCluster having the expected condition (expected: %+v vs actual: %+v)", name, condition, cluster.Status.Conditions)
```